src/UI/Components/ServiceInstanceForm/Helpers/createFormState.ts

```diff
@@ -173,9 +194,22 @@ export const createDuplicateFormState = (
         case "Textarea":
         case "TextList":
         case "Text": {
-          acc[curr.name] = curr.type.includes("dict")
-            ? stringifyDict(originalAttributes?.[curr.name])
-            : cloneDeep(originalAttributes?.[curr.name]);
+          if (curr.type === "int?" || curr.type === "float?") {
```

Comment:
This feels redundant, and not optimized. 
The formState is a complex function, adding even more complexity will make it hard to read.